bundle/rhpam/manifests/bamoe-kogito-operator.clusterserviceversion.yaml

```diff
@@ -983,8 +981,8 @@ spec:
   - native
   - automation
   labels:
-    alm-owner-kogito: rhpam-kogito-operator
-    operated-by: rhpam-kogito-operator.8.0.0-1
+    alm-owner-kogito: bamoe-kogito-operator
+    operated-by: bamoe-kogito-operator.0.0.0
```

Comment:
0.0.0 ?

Reply:
it is a place holder, it gets updated when you run make bundle.